cmd/crc/cmd/start.go

```diff
@@ -263,15 +255,19 @@ type templateVariables struct {
 }
 
 func writeTemplatedMessage(writer io.Writer, s *startResult) error {
-	if s.ClusterConfig.ClusterType == preset.OpenShift {
+	if s.ClusterConfig.ClusterType == preset.OpenShift || s.ClusterConfig.ClusterType == preset.OKD {
 		return writeOpenShiftTemplatedMessage(writer, s)
 	}
 
 	return writePodmanTemplatedMessage(writer, s)
 }
 
 func writeOpenShiftTemplatedMessage(writer io.Writer, s *startResult) error {
-	parsed, err := template.New("template").Parse(startTemplateForOpenshift)
+	tmpl := startTemplateForOpenshift
+	if s.ClusterConfig.ClusterType == preset.OKD {
+		tmpl = startTemplateForOKD
+	}
```

Comment:
The previous code was first printing the OpenShift start template, and then appending a warning after it when OKD is in use. This seems to have been lost here, and to only print the warning for OKD?

Reply:
Ah right, will update 👍🏼